package-lock.json

```diff
@@ -38422,7 +38422,7 @@
     "select": {
       "version": "1.1.2",
       "resolved": "https://registry.npmjs.org/select/-/select-1.1.2.tgz",
-      "integrity": "sha1-DnNQrN7ICxEIUoeG7B1EGNEbOW0="
+      "integrity": "sha512-OwpTSOfy6xSs1+pwcNrv0RBMOzI39Lp3qQKUTPVVPRjCdNa5JH/oPRiqsesIskK8TVgmRiHwO4KXlV2Li9dANA=="
```

Comment:
This change is unneeded too

Reply:
I use npm `6.14.14` and also did delete the node_modules before installing, but still changing that integrity 🤔

Reply:
Since it's only one change, you can manually remove it 😄😏